apply/processor/gen.go

```diff
@@ -88,10 +91,36 @@ func (g *GenerateProcessor) GetPipeLine() ([]func(cluster *v2.Cluster) error, er
 		g.MountRootfs,
 		g.ApplyRegistry,
 		g.UnmountImage,
+		g.CmdAddRegistryHosts,
 	)
 	return todoList, nil
 }
 
+func (g *GenerateProcessor) CmdAddRegistryHosts(cluster *v2.Cluster) error {
```

Comment:
Please add some comments for this function. This could bring more code readability.